pkg/github/github.go

```diff
@@ -686,6 +712,37 @@ func (g *GitHub) UploadReleaseAsset(
 	return asset, nil
 }
 
+// ToRequest builds an issue request from the set of options
+func (nio *NewIssueOptions) toRequest() *github.IssueRequest {
+	request := &github.IssueRequest{}
+	request.Assignee = &nio.Assignee
+	if nio.State == "open" || nio.State == "closed" || nio.State == "all" {
+		request.State = &nio.State
+	}
+
+	if len(nio.Labels) > 0 {
+		request.Labels = &nio.Labels
+	}
+
+	if len(nio.Assignees) > 0 {
+		request.Labels = &nio.Assignees
```

Comment:
This seems wrong. Should we really assign assignees to labels?

Reply:
Good catch. Fixed :+1: